### ISSUE TITLE
How to configure Ruff to disallow double assignments in single lines in Python?

### ISSUE THREAD

**Issue by javinba** [astral-sh/ruff#11317]
I'm looking for a way to configure Ruff to disallow double assignments in single lines, like this:

```python
name, position = files.name, pos
```
Instead, I want the linter to enforce separate assignments on separate lines, like this:
```python
name = files.name
position = pos
```
However, I still want to allow tuple unpacking, like this:
```
name, position = extract_details()
```
Is there a way to configure Ruff to achieve this behavior?

**Comment by MichaReiser** [comment]
I don't think such a rule exists today. I just tried to see if Ruff flags the assignment when enabling all rules but it doesn't. 

https://play.ruff.rs/e1feb7f2-7c32-4bfd-ae87-b12102a382a4

**Comment by javinba** [comment]
Hi @MichaReiser , thanks for replying. Do you think it would be possible to add this rule? :slightly_smiling_face:

**Comment by harupy** [comment]
+1

**Comment by MichaReiser** [comment]
What would the logic for a rule like this. E.g would the following be allowed?

```python
b = [1, 2]
name, position, b = *b, 2
```

**Comment by harupy** [comment]
I think that would be allowed.
